integration/hsm/helpers.go

```diff
@@ -275,7 +275,7 @@ func newAuthConfig(t *testing.T, log utils.Logger) *servicecfg.Config {
 	}
 	var err error
 	config.Auth.ClusterName, err = services.NewClusterNameWithRandomID(types.ClusterNameSpecV2{
-		ClusterName: "testcluster",
+		ClusterName: "test-cluster",
```

Comment:
this just matches the cluster name used in lib/auth/keystore tests so that it's easier to find all the test keys in e.g. the AWS console